Cargo.toml

```diff
@@ -108,6 +111,7 @@ saffron = { path = "./poly-commitment", version = "0.1.0" }
 signer = { path = "./signer", version = "0.1.0" }
 turshi = { path = "./turshi", version = "0.1.0" }
 utils = { path = "./utils", version = "0.1.0" }
+wasm-types = { path = "./crates/wasm-types" }
```

Comment:
Is there a goal to move all WebAssembly definitions also present in `kimchi` in this crate?